src/components/layout/UpgradeTier/UpgradeTier.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+
+import { NAME_PRICE_CALC, NAME_PRICE_INFO } from '../../../../types/constants';
+import { useStateValue } from '../../../state/state';
+import TierCard from '../../cards/TierCard/TierCard';
+import { AlertCircle } from '../../icons';
+import YearsCounter from '../../inputs/YearsCounter/YearsCounter';
+import './styles.css';
+
+function UpgradeTier({ domain }: { domain: string | undefined }) {
+  const [{ arnsSourceContract }] = useStateValue();
+  // name is passed down from search bar to calculate price
+  const [tier, setTier] = useState(1);
+  const [price, setPrice] = useState(0);
+  const [years, setYears] = useState(1);
+  const [priceInfo, setPriceInfo] = useState(false);
+
+  useEffect(() => {
+    const fees = arnsSourceContract.fees;
+    setPrice(NAME_PRICE_CALC({ domain, tier, years, fees }));
+  }, [years, tier, domain, arnsSourceContract]);
+
+  return (
+    <div className="upgradeTier">
+      <YearsCounter setCount={setYears} count={years} />
+      <div className="cardContainer">
+        <TierCard thisTier={1} setTier={setTier} tier={tier} />
+        <TierCard thisTier={2} setTier={setTier} tier={tier} />
+        <TierCard thisTier={3} setTier={setTier} tier={tier} />
```

Comment:
i think `selectedTier` and `tier` are a little more clear
```suggestion 
      // TODO: pul tiers from contract when evolved
       Object.keys(TIER_DATA).map((tier) => {
            return  <TierCard tier={tier} setTier={setTier} selectdTier={selectedTier} />
```